app/config/config_test.yml

```diff
@@ -4,6 +4,7 @@ parameters:
 
 imports:
     - { resource: config_dev.yml }
+    - { resource: "../../tests/Resources/config/services.yml" }
```

Comment:
Weird to load services here :thinking: 
You should edit the `services_test.yml` instead :)

Reply:
If you refer to files like `config/services/admin/services_test.yml` they are not related to the Framework, they are used by our LegacyContainer We don't have any files specific for test services for now (they were defined directly in the config file up until now)

And in Symfony 3.4 there is not automatic structure for them either